src/Grpc/HttpApi/test/Microsoft.AspNetCore.Grpc.Swagger.Tests/Microsoft.AspNetCore.Grpc.Swagger.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
1. This setting isn't handled correctly for `$(NoWarn)` because non-warnings should be preserved
2. The root Directory.Build.props file includes `CS1591` for all test projects so this doesn't help anyway